code/modules/surgery/organs/stomach/stomach_ethereal.dm

```diff
@@ -6,11 +6,34 @@
 	var/crystal_charge = ETHEREAL_CHARGE_FULL
 	///used to keep ethereals from spam draining power sources
 	var/drain_time = 0
+	var/damagemodifer = 0.5
+	low_threshold_passed = "<span class='info'>You can feel your power draining as a feeling of pins and needles fills your abdomen.</span>"
+	high_threshold_passed = "<span class='warning'>Your abdomen is filled with the feeling of shocks and electric discharges!</span>"
+	high_threshold_cleared = "<span class='info'>The electric feeling in your stomach seems to fade away, leaving behind a feeling of pins and needles</span>"
+	low_threshold_cleared = "<span class='info'>The pins and needles feeling seems to die out as your charge starts to feel far more stable.</span>"
 
 /obj/item/organ/stomach/ethereal/on_life(delta_time, times_fired)
 	. = ..()
 	adjust_charge(-ETHEREAL_CHARGE_FACTOR * delta_time)
 	handle_charge(owner, delta_time, times_fired)
+	if(damage > maxHealth)//cap battery damage
+		damage = maxHealth
+	if(damage < low_threshold)
+		return
+	adjust_charge(-0.8 * (damage / 5) * delta_time) //at 100 damage ethereals will loose 16 charge per second
+	handle_charge(owner, delta_time, times_fired)
+	var/mob/living/carbon/organ_owner = owner
+	if (prob(7))
+		to_chat(owner, "<span class='warning'>You feel a jolt of elecricity from your abdomen!</span>")
+	if(!(organ_flags & ORGAN_FAILING))
+		return
+	if(organ_owner)
+		organ_owner.adjustToxLoss(0.65 * delta_time, TRUE, TRUE)
```

Comment:
this damage should be capped like the damage from having low charge is

Reply:
no, that is kinda the point, it is supposed to be fatal at failure

Reply:
the damage cap for being low on charge is kind of pointless if you can die from being low on charge anyway.

also, no other species outright dies from (just) being low on hunger; why should ethereals?

Reply:
The uncapped damage only happens if the organ is failing, not under normal conditions

Reply:
> The uncapped damage only happens if the organ is failing, not under normal conditions

yeah, but the organ can fail from the damage from being low on charge

Reply:
you need to, wait to get to empty charge, then wait another 5 minutes, i wanted ethereals to have more danger in going with zero charge for extended periods of time